cmd/scaffold.go

```diff
@@ -101,9 +103,39 @@ var dockerfileCmd = &cobra.Command{
 			})
 		}
 
+		hasVars := len(manifest.Variables) > 0
+		var runtimeConfigDest string
+		if hasVars {
+			lgr.Debug("found variables, writing runtime config")
+
+			// check if runtime config already exists
+			runtimeConfigDest = filepath.Join(filepath.Dir(spinManifest), "spin-config.toml")
+			if _, err := os.Stat(runtimeConfigDest); err == nil && !override {
+				return usererror.New(
+					errors.New("file exists"),
+					fmt.Sprintf("File %s already exists. Use --override to overwrite.", runtimeConfigDest),
+				)
+			}
+
+			runtimeConfig := spin.NewRuntimeConfig(manifest)
```

Comment:
correct to make new or should we be merging with existing one? enforcing only that the fields we control are in?